Wikibase.php

```diff
@@ -260,6 +276,22 @@
 		'jsonld',
 	];
 	$wgWBRepoSettings['taintedReferencesEnabled'] = true;
+	$wgWBRepoSettings['entitySources'] = [
```

Comment:
> This can mostly all be configured in ManageWiki.

hmm? how?

Reply:
This isn't available anywhere in ManageWiki

Reply:
> `$wmgWikibaseRepoUrl`, `$wmgWikibaseRepoDatabase`, `$wmgWikibaseRepoPropertyNamespaceID`, and `$wmgWikibaseRepoItemNamespaceID` configures entitySources for both client and repo it seems.

Ah, no, that's a different case. I could also set it to those `wmg`s that you listed
If you look closely there are some things I set that aren't default.

Reply:
interwikiPrefix is the only other one I noticed to be different, which that one can probably be added to ManageWiki also.

Reply:
Lexeme is the same as the default, and property namespace can be changed with $wmgWikibaseRepoPropertyNamespaceID

Reply:
The point is, 'lexeme' is not explicitly mentioned under the entityNamespace in entity sources, that's where I have a problem with all the default configs

Reply:
It's mentioned below it.
https://github.com/miraheze/mw-config/blob/54ac84d78d4d706e4075109569fde522f539d3f7/Wikibase.php#L24

Reply:
> It's mentioned below it.
> 
> https://github.com/miraheze/mw-config/blob/54ac84d78d4d706e4075109569fde522f539d3f7/Wikibase.php#L23-L26

Why don't we clean it up from there and add it together with its mate :)

Reply:
Because we can't. That one can only be set when WikibaseLexeme is enabled. Which is why it's different.

Reply:
I see

Reply:
So how do we do the interwiki Prefix?

Reply:
Another thing, I am setting as `gratisdata` and not `local`.
And the default is local

Reply:
Is there a reason for that? If so just set gratisdata to the local one, for example:
```php
$wgWBRepoSettings['entitySources']['gratisdata'] = $entitySources['local'];
$wgWBRepoSettings['entitySources']['gratisdata']['interwikiPrefix'] = 'd';
```

But only if a valid reason for doing so please.

Reply:
Yeah, mainly for future reference, when I want to set up entity sources for gpcommonswiki's Wikibase MediaInfo, I'd also need to set as `gpcommons` so that we don't have two `local`s with different entity sources. 
So that `itemAndPropertySource` can be set accordingly (gratisdata and gpcommons).

Reply:
Ok, please do what I mentioned above then.

Reply:
> Ok, please do what I mentioned above then.

Done